sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -217,6 +217,38 @@ public partial class CassandraTableThroughputSettingResource : Azure.ResourceMan
+        public virtual Azure.Response<bool> Exists(string clientEncryptionKeyName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<bool>> ExistsAsync(string clientEncryptionKeyName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<Azure.ResourceManager.CosmosDB.ClientEncryptionKeyGetResultResource> Get(string clientEncryptionKeyName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Pageable<Azure.ResourceManager.CosmosDB.ClientEncryptionKeyGetResultResource> GetAll(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.CosmosDB.ClientEncryptionKeyGetResultResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.CosmosDB.ClientEncryptionKeyGetResultResource>> GetAsync(string clientEncryptionKeyName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.CosmosDB.ClientEncryptionKeyGetResultResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.CosmosDB.ClientEncryptionKeyGetResultResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.CosmosDB.ClientEncryptionKeyGetResultResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.CosmosDB.ClientEncryptionKeyGetResultResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class ClientEncryptionKeyGetResultData : Azure.ResourceManager.Models.ResourceData
+    {
+        public ClientEncryptionKeyGetResultData() { }
+        public Azure.ResourceManager.CosmosDB.Models.ClientEncryptionKeyGetPropertiesResource Resource { get { throw null; } set { } }
+    }
+    public partial class ClientEncryptionKeyGetResultResource : Azure.ResourceManager.ArmResource
```

Comment:
This is really a bad name for a resource.
Does this request path `/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/sqlDatabases/{databaseName}/clientEncryptionKeys/{clientEncryptionKeyName}` really correspond to a resource?
If so, we should rename the class `ClientEncryptionKeyGetResultData` to `CosmosDBSqlDatabaseClientEncryptionKey`, and these three class will automatically rename to follow the pattern of `XXXCollection + XXXData + XXXResource`

Reply:
@kr-santosh please take a look at this.

Reply:
I am ok with renaming as suggested

Reply:
isnt this auto generated from swagger spec? But the swagger PR was approved and merged? Do we manually change this? Am confused @kshittiz7 @ArcturusZhang

Reply:
This was originally public

 partial class ClientEncryptionKeyGetResults : ARMProxyResource 

i guess data was appended to it.

This is a database child resource

Reply:
Does using rename-model: suffice. Do have a look at the changes made.

Reply:
#resolved